lstn/templates/index.html

```diff
@@ -91,17 +91,19 @@
 
   <div id="content" class="container" data-ng-view></div>
   <div id="apiswf"></div>
-
+  
+  <script src="//twemoji.maxcdn.com/twemoji.min.js"></script>
```

Comment:
Yep, we need it. The bower include is for an angular library that wraps this JS lib.